backend/modules/prices/schemas.py

```
@@ -10,3 +11,15 @@ class InstrumentPriceSchema(BaseModel):
     high: Decimal
     low: Decimal
     close: Decimal
+
+
+type PriceUpdateHandler = Callable[[dict[str, float]], None]
```

Pylint (3.3.7)

[error] Parsing failed: 'invalid syntax (modules.prices.schemas, line 16)'

(E0001)